packages/@ember/-internals/glimmer/lib/helpers/internal-helper.ts

```diff
@@ -1,7 +1,10 @@
 import type { InternalOwner } from '@ember/-internals/owner';
 import type { Helper, HelperDefinitionState } from '@glimmer/interfaces';
-import { setInternalHelperManager } from '@glimmer/manager';
+// import { setInternalHelperManager } from '@glimmer/manager';
```

Comment:
why comment this out?

If we need to re-define the implementation, I think that could happen in this PR (like, if it's copying from glimmer-next, for example)